### DISCUSSION TITLE
How can I go to the Editing Mode directly?

### DISCUSSION THREAD

**Discussion by ghost** [redeclipse/base#1217] [Help and Support]
If I select "Editing Mode" and a map, the "Start Game" button can't be clicked, it only works when I select other modes like **Capture** or **Bomber**(except **Edit** and **Demo**).

**Comment by ghost** [comment]
I run 2.0.0 version(build 2978) of the game on win 8.1 pro x64.

**Comment by Jigoku** [comment]
You can type `/edit mapname`, if the map doesn't already exist it will load a new/empty map for you.

**Reply by dc-redeclipse** [reply]
maybe a button "create new map" in hud for this would be helpfully for new players.

**Comment by qreeves** [comment]
Transferring to world as this is not a bug report.
